index.md

```diff
@@ -40,6 +40,7 @@ SecTalks is organised in the following cities. Visit each city page to know more
 * [Melbourne, Australia](./melbourne)
 * [London, UK](./london)
 * [Canberra, Australia](./canberra)
+* [Gold Coast, Australia](./goldcoast)
```

Comment:
@f3rn0s This list is sorted based on when each city had their SecTalks established. Please move this to the end of the list.